recipe/meta.yaml

```
@@ -7,7 +7,7 @@ package:
 
 source:
   url: https://pypi.io/packages/source/d/dask/dask-{{ version }}.tar.gz
```

Suggesting temporarily using the hashed link while the upstream issue with the redirect one ( https://github.com/pypi/warehouse/issues/11934 ) gets fixed

```suggestion
  url: https://files.pythonhosted.org/packages/68/e1/1edb5eeb15f88267075c37ee8c83c4527e462bdea328566c5b1e88db705c/dask-2022.7.1.tar.gz
```